pkg/sdkserver/sdkserver_test.go

```diff
@@ -763,8 +758,6 @@ func TestSDKServerSendGameServerUpdate(t *testing.T) {
 
```

Comment:
On line 756 please adjust the code comment to remove the "if SDKWatchSendOnExecute feature is turned on," part (and then capitalize the beginning of the sentence).